### PR TITLE
Fix scroll in browser

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue #537 .

### Summary of Changes
Fixes the scroll bar in image browser.

### CODE REVIEW

**Comment by laoneo** [comment]
Can you fix conflicts here?

**Comment by laoneo** [comment]
When the browser window has a scroll bar (eg. in debug mode), then I have with this patch two scroll bars. Can it be changed the way that only one scroll bar will appear?

**Comment by ciar4n** [comment]
Just removed the min height which resolves this issue.

File: administrator/components/com_media/resources/styles/components/_layout.scss [hunk 1]
```diff
@@ -34,7 +34,6 @@
 .media-main {
   position: relative;
   height: calc(100vh - 160px);
```
Comment (laoneo) [inline administrator/components/com_media/resources/styles/components/_layout.scss:36]: It still does not work, but when I remove this line then it is ok.

Comment (ciar4n) [reply administrator/components/com_media/resources/styles/components/_layout.scss:36]: Ok. I see now. The issue is only present in debug mode.

Removing this line means the grid/list area does not scroll. All images are displayed and instead you use the default browser scroll. Originally  the idea to adding the scroll to the grid/list area was so you can scroll the images while still keeping the toolbar and sidebar displayed. Should I still remove it?

Comment (laoneo) [reply administrator/components/com_media/resources/styles/components/_layout.scss:36]: You are the CSS master. I do just report what I see.

**Comment by laoneo** [comment]
@ciar4n can you have another look on this pr. Something screwed up here.

**Comment by ciar4n** [comment]
The SCSS compiler is only compiling to mediamanager.css however the media manager is loading mediamanager.min.css ?

Not the problem here but an issue I face when I attempt to fix this.

**Comment by ciar4n** [comment]
All should be good now. 

There is a scroll bar on both the media browser (presuming enough items) and the viewport when debug is enabled but there is no sensible way to avoid that.

**Comment by laoneo** [comment]
I still have two scroll bars
image

**Comment by ciar4n** [comment]
Have you debug enabled?

**Comment by laoneo** [comment]
Yes. Does that mather?

**Comment by ciar4n** [comment]
It makes a difference here...

> There is a scroll bar on both the media browser (presuming enough items) and the viewport when debug is enabled but there is no sensible way to avoid that.

I could work on some sort of a solution however to me it is a bit of an edge case?

**Comment by laoneo** [comment]
Ah no, when it is only when debug is enabled, then all is good. Now we need to fix the drone tests too.
